RecoTauTag/TauTagTools/interface/ElementsInEllipseRef.h

```diff
@@ -1,26 +0,0 @@
-#ifndef ElementsInEllipseRef_h
-#define ElementsInEllipseRef_h
-
-#include "RecoTauTag/TauTagTools/interface/Ellipse.h"
-#include "DataFormats/Common/interface/RefVector.h"
-#include <utility>
-
-template  <typename T, typename C>
-  class ElementsInEllipse{
```

Comment:
@mbluj @steggema @swozniewski 
please check and confirm that this is not needed anymore.
It looks like this was (re)added in 2013 for HLT compatibility 
https://github.com/cms-sw/cmssw/commit/0f45b619a83c7358addd160bfabe4f2534848054#diff-b027deb3da39d07e90367d5113d72d58

Reply:
It indeed looks as pretty old, unused code - @veelken do you remember what the code was used for (for TaNC?) and if it can be safely removed?

Reply:
I can confirm that this is on the list of files that we're considering for removal in an upcoming cleanup, and also that it is not used by current Tau HLT algorithms (and I don't think we intend to run Run-1 tau HLT algorithms with future CMSSW).